frontend/src/views/dev/Performance.vue

```diff
@@ -1,24 +1,19 @@
 <template>
-  <v-container fluid>
-    <content-card title="Config">
-      <v-card-text>
-        <table class="result-table">
-          <tr>
-            <th>Endpoint</th>
-            <th>Collection ms</th>
-            <th>Entity ms</th>
-          </tr>
-          <tr v-for="result in results" :key="result.endpoint">
-            <td>{{ result.endpoint }}</td>
-            <td class="text-right">
-              {{ result.collection }}
-            </td>
-            <td class="text-right">
-              {{ result.entity }}
-            </td>
-          </tr>
-        </table>
-      </v-card-text>
+  <v-container fluid style="max-width: 1024px">
+    <content-card title="Performance" toolbar>
+      <v-data-table
```

Comment:
I think v-data-table is limited to show 10 items by default. You probably want `:items-per-page="-1"`

Reply:
Thanks! done in 721a8d0